packages/cli/src/commands/build-docs/index.ts

```diff
@@ -0,0 +1,37 @@
+import { loadAndBundleSpec } from 'redoc';
+import { dirname, resolve } from 'path';
+import { writeFileSync, mkdirSync } from 'fs';
+
+import { getObjectOrJSON, handleError, isURL, getPageHTML } from './utils';
+import type { BuildDocsArgv } from './types';
+
+export const handlerBuildCommand = async (argv: BuildDocsArgv) => {
+  const config = {
+    output: argv.o,
+    cdn: argv.cdn,
+    title: argv.title,
+    disableGoogleFont: argv.disableGoogleFont,
+    templateFileName: argv.template,
+    templateOptions: argv.templateOptions || {},
+    redocOptions: getObjectOrJSON(argv.options),
+  };
+
+  const redocCurrentVersion = require('../../../package.json').dependencies.redoc;
+  const pathToSpec = argv.spec;
+
+  try {
+    const start = Date.now();
+    const spec = await loadAndBundleSpec(isURL(pathToSpec) ? pathToSpec : resolve(pathToSpec));
+    const pageHTML = await getPageHTML(spec, pathToSpec, { ...config, redocCurrentVersion });
+
+    mkdirSync(dirname(config.output), { recursive: true });
+    writeFileSync(config.output, pageHTML);
+    const sizeInKiB = Math.ceil(Buffer.byteLength(pageHTML) / 1024);
+    const time = Date.now() - start;
```

Comment:
Instead of this I would recommend to use `getExecutionTime` from `packages/cli/src/utils.ts`